test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -409,6 +599,36 @@ public void testBlobColumnModeContentTypeCol() throws Exception {
   }
 
   @Test
+  public void testBlobColumnModeContentTypeCol_nullValue() throws Exception {
```

Comment:
These are also not per-type code. Also, maybe a naming convention to distinguish these from the test<Mode> tests, like testConfigContentTypeCol_null?